platform/util/platform-util/src/test/java/org/codice/ddf/platform/util/InputValidationTest.java

```diff
@@ -95,4 +114,30 @@ public void testCheckForClientSideVulnerableMimeTypeGood() {
         boolean result = InputValidation.checkForClientSideVulnerableMimeType(GOOD_MIME);
         assertTrue(result);
     }
+
+    @Test
+    public void testSanitizeFilenameKnownBadCaseInsensitive() {
+        String sanitizedName = InputValidation.sanitizeFilename(BAD_FILE3);
+        assertThat(sanitizedName, is(DEFAULT_FILE));
```

Comment:
✏️ Perhaps use the Hamcrest `assertThat` method which takes a String reason and the boolean check to help provide better logging in case of failure. Same with the other tests.